smokes/e2e/home.scenarios.coffee

```diff
@@ -0,0 +1,42 @@
+# coffee script
+# test goal: checks the the number of element present in home page
+# to test this part: two different builds need to be started
+
+
+forcePage = require('./force.coffee')
+builderPage = require('./builder.coffee')
+homePage = require('./home.coffee')
+
+
+describe('', () ->
+    force = null
+    builder = null
+    home = null
+
+    beforeEach(() ->
+        builder = new builderPage('runtest', 'force')
+        force =  new forcePage()
+        home = new homePage()
+        builder.goDefault()
+    )
+
+    describe 'manage home web page', () ->
+        it 'should go to the home page and check the different builder', () ->
+            researchData = {
```

Comment:
researchData? just call it builderNames. I dont think this is stable to hardcode the order like this.
I think the home page builder order is undefined. We should define it to order it by buildername.
This will make the test more stable to environement changes